store/src/java-test/com/zimbra/cs/filter/NotifyTest.java

```diff
@@ -112,4 +112,37 @@ public void filterValidToField() {
 
     }
 
+    @Test
+    public void testNotifyMailtoWithHeaderVariable() {
+        String sampleMsg = "from: abc@zimbra.com\n"
+                + "Subject: Hello\n"
+                + "to: test@zimbra.com\n";
+        String filterScript = "require [\"enotify\", \"variables\"];\n"
+            + "set \"to\" \"nick\";\n"
+            + "if anyof (header :contains [\"Subject\"] \"Hello\") {\n"
+            + "notify \"test2@zimbra.com\" \"\" \"${SUBJECT} ${to}\"\n"
```

Comment:
But this implementation brakes the backward compatibility.  In the context of the legacy notification action, the value of the ${SUBJECT}, ${TO}, ${CC}, ${BODY}, etc. should be referred from the triggering message.  I think the RFC 5229 complied variable should be applied only to the RFC compiled notification action, not to the legacy version.